protocol-units/da-sequencer/node/src/whitelist.rs

```diff
@@ -0,0 +1,99 @@
+use std::{
+	collections::HashSet,
+	fs,
+	path::PathBuf,
+	sync::{Arc, RwLock},
+	thread,
+	time::Duration,
+};
+
+use ed25519_dalek::VerifyingKey;
+use hex::FromHex;
+
+#[derive(Clone)]
+pub struct Whitelist {
+	inner: Arc<RwLock<HashSet<VerifyingKey>>>,
+}
+
+impl Whitelist {
+	pub fn load(path: &PathBuf) -> std::io::Result<HashSet<VerifyingKey>> {
```

Comment:
Put the `load` and `start_reload_thread` private they aren't call outside the file.